sample/build.gradle

```diff
@@ -22,6 +22,13 @@ android {
   }
 }
 
+// Disable unnecessary release build type
+androidComponents {
+  beforeVariants(selector().all()) {
```

Comment:
Would it make sense to encode the comment in code?
```
beforeVariants(selector().withBuildType("release")) { enable = false }
```
```

Reply:
I tested mayny things with the Groovy API and forgot to actually make it straightforward 🙈 
Thanks!